modules/katamari/src/test/ts/atomic/api/arr/ArrFromTest.ts

```diff
@@ -2,7 +2,7 @@ import * as Arr from 'ephox/katamari/api/Arr';
 import { UnitTest, Assert } from '@ephox/bedrock-client';
 
 UnitTest.test('ArrFromTest', () => {
-  const func = function (...x: any[]) {
+  const func = function (..._x: any[]) {
     Assert.eq('eq', [ 1, 2, 3 ], Arr.from(arguments));
```

Comment:
Change this to use x instead of arguments

Reply:
Scratch that, we do probably want to be using `arguments` here as it's testing converting a non-array to an array...